packages/components/src/Base/FC__E.re

```diff
@@ -105,6 +118,31 @@ module L = {
   let update = Rationale.RList.update;
   let iter = List.iter;
   let findIndex = Rationale.RList.findIndex;
+  module React = {
+    let map = (map: 'a => React.element, list: list('a)): React.element => {
+      let a = [||];
+      let rec loop = list =>
+        switch (list) {
+        | [first, ...rest] =>
+          let _ = Js.Array.push(map(first), a);
```

Comment:
why do you need to use Js.Array.push? Can you use some other append method? Can you make sure it's part of E.A instead of calling it directly?